src/test/java/com/google/sps/data/DatastoreEntityToObjectConverterTest.java

```diff
@@ -97,7 +97,7 @@ public void setUp() {
         reactions.put(Reaction.CRYING, (long) 0);
         reactions.put(Reaction.HEART, (long) 0);
         reactions.put(Reaction.LAUGHING, (long) 0);
-        this.post = new Post(commentQuestion, commentAnswer, replies, tab.getTabName(), postId, reactions); 
+        this.post = new Post(commentQuestion, commentAnswer, replies, tab.getTabName(), postId, reactions, 0); 
```

Comment:
what variable is 0? prefer doing something like ..., reactions, /* varName = */ 0